src/Dispatcher.ts

```diff
@@ -9,7 +9,6 @@ import {ActionMode} from './Actions/Mode';
 import {ActionMoveCursor} from './Actions/MoveCursor';
 
 export class Dispatcher {
-
```

Comment:
We are leaving one empty line between class statement and the first member statement in this project.